### DISCUSSION TITLE
Create issue by cmd+enter will loss content

### DISCUSSION THREAD

**Discussion by li-yechao** [community/community#77065] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Demo repo: https://github.com/li-yechao/test-sync/issues

Steps:

- Create an issue in a repo with issue template
- Input title
- Input some text in the first textarea of the template
- Cmd+Enter
- The content of the first textarea lost

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by oliviertassinari** [comment]
I have been hit at least 8 times by this bug in the last 3 months. Please fix this! It feels like this was working correctly in the past, I suspect a regression.
